Source/CombatExtended/CombatExtended/Jobs/JobDriver_Hunt.cs

```diff
@@ -172,18 +172,17 @@ Toil GotoCastPosition(TargetIndex targetInd, bool closeIfDowned = false)
                 Job curJob = actor.CurJob;
                 Thing thing = curJob.GetTarget(targetInd).Thing;
                 var pawnVictim = thing as Pawn;
-                IntVec3 intVec;
+
+                
```

Comment:
Looks fine other than this.  Try to keep to 2 or fewer blank lines unless there's a strong reason to visually offset a block.

Reply:
done